### PR TITLE
Gun Buffs

### PR DESCRIPTION
**Pull request by TwoDucksOnnaPlane**
# Description

Buffs projectile weapons and gives uranium rounds ignoreResistances.
This is 1000% a mald PR after getting killed while using a minigun.

---

# Changelog

:cl:
- add: Uranium rounds now have Armor Piercing.
- tweak: Most bullets have been buffed.

## Summary by CodeRabbit

- **New Features**
	- Increased damage values for various bullet types across multiple weapon categories, enhancing their effectiveness in gameplay.
	- Added a new property, `ignoreResistances`, to certain bullet entities to improve their performance against resistant targets.

- **Bug Fixes**
	- Adjusted damage values for bullets to ensure consistency and balance in gameplay.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml [hunk 1]
```diff
@@ -8,6 +8,6 @@
     damage:
       types:
         Piercing: 40
-        Structural: 30
+        Structural: 150
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml:11]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/heavy_rifle.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 19
+        Piercing: 24
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/heavy_rifle.yml:10]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/heavy_rifle.yml [hunk 3]
```diff
@@ -18,4 +18,4 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 5
+        Piercing: 24
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/heavy_rifle.yml:21]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml [hunk 4]
```diff
@@ -7,7 +7,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 19
+        Piercing: 24
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml:10]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml [hunk 5]
```diff
@@ -40,8 +40,8 @@
   - type: Projectile
     damage:
       types:
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml:42]: Add a `# Spacious Station - buff Blunt/Heat damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/pistol.yml [hunk 6]
```diff
@@ -7,7 +7,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 16
+        Piercing: 20
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/pistol.yml:10]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/pistol.yml [hunk 7]
```diff
@@ -41,7 +41,7 @@
     damage:
       types:
         Blunt: 2
-        Heat: 14
+        Heat: 18
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/pistol.yml:44]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/rifle.yml [hunk 8]
```diff
@@ -7,7 +7,7 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 17
+        Piercing: 22
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/rifle.yml:10]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/rifle.yml [hunk 9]
```diff
@@ -41,8 +41,8 @@
     damage:
       types:
         Blunt: 2
-        Heat: 15
-  
+        Heat: 20
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/rifle.yml:44]: Add a `# Spacious Station - buff damage` comment to this line

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/rifle.yml [hunk 10]
```diff
@@ -52,6 +52,7 @@
   - type: Projectile
```
Comment (angelofallars) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/rifle.yml:52]: Add a `# Spacious Station - buff damage and ignore resistances` comment to this line

**Review by angelofallars — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/heavy_rifle.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (TwoDucksOnnaPlane) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/heavy_rifle.yml:21]: yes, I know it ups the DPS coderabbit
that is the point as the minigun is an admin only weapon and nothing else fires the rounds

**Review by angelofallars — APPROVED** [review]

**Pull request merged**